src/explore-education-statistics-common/src/modules/find-statistics/components/DataBlockTabs.tsx

```diff
@@ -84,39 +132,76 @@ const DataBlockTabs = ({
             }
             title="Chart"
           >
-            {!!error && errorMessage}
+            {(!!tableDataError || !!geoJsonError) && errorMessage}
 
             {fullTable && (
               <ErrorBoundary fallback={errorMessage}>
                 {dataBlock.charts.map((chart, index) => {
                   const key = index;
 
-                  const axes = { ...chart.axes } as Required<AxesConfiguration>;
+                  const rendererProps: Omit<ChartRendererProps, 'chart'> = {
+                    id: `${id}-chart`,
+                    source: dataBlock?.source,
+                  };
 
                   if (chart.type === 'infographic') {
                     return (
                       <ChartRenderer
-                        {...chart}
-                        id={`${id}-chart`}
+                        {...rendererProps}
                         key={key}
-                        axes={axes}
-                        data={fullTable?.results}
-                        meta={fullTable?.subjectMeta}
-                        source={dataBlock?.source}
-                        getInfographic={getInfographic}
+                        chart={{
+                          ...chart,
+                          data: fullTable?.results,
+                          meta: fullTable?.subjectMeta,
```

Comment:
`fullTable` should be defined so don't need optional chaining. Same applies in other places below as well.